fhirResourcesToLoad/o2jane_04__device-request-a4217.json

```diff
@@ -0,0 +1,41 @@
+{
+    "resourceType": "DeviceRequest",
+    "id": "devreq16-a4217",
```

Comment:
These elements are required: DeviceRequest.basedOn, DeviceRequest.instantiatesCanonical

Reply:
not sure if I should add these and if so where?

Reply:
Please check other DeviceRequest resources